### PR TITLE
feat(render): add multi state + empty placeholders

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to https://github.com/bpmn-io/properties-panel/issues/69

```sh
npx @bpmn-io/sr "bpmn-io/dmn-js-properties-panel#update-placholders"
```

image

----

What's missing here are proper icons, but I think the texts are already worth moving forward.

### CODE REVIEW

**Comment by MaxTru** [comment]
> To consider here, too: [bpmn-io/bpmn-js-properties-panel#687 (comment)](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/687#issuecomment-1132684836)

@pinussilvestrus will you adjust that? I support the comment and think a simpler message would make sense

**Comment by pinussilvestrus** [comment]
This PR is already aligned with what we have merged in the bpmn-js-properties-panel 👍

File: test/TestHelper.js [hunk 1]
```diff
@@ -91,6 +93,26 @@ export function insertCoreStyles() {
   );
 }
 
+/**
+ * Execute test only if currently installed @bpmn-io/properties-panel is of given version.
```
Comment (MaxTru) [inline test/TestHelper.js:97]: Nice

**Review by MaxTru — APPROVED** [review]
Mostly the same as for bpmn-js-..... I like how you use semver for the integration test determination

**Pull request merged**